smart-contracts/contracts/burn_coins/src/state.rs

```diff
@@ -0,0 +1,30 @@
+use cosmwasm_std::Coin;
+use cw_storage_plus::Item;
+
+pub const AMOUNT_BURNT: Item<Vec<Coin>> = Item::new("burnt_coins");
```

Comment:
Since total burnt here is an item and a `Vec` could theoretically contain multiple coins, meaning that the entire Vec would have to be loaded for us to update one coin. Cleaner would be to use a `Map`, with the denom as key and the value as the amount